code/script.js

```diff
@@ -291,54 +282,44 @@ const selectQuestion = () => {
         category
       }
   } else if (category === 'hat') {
-    currentQuestion = {
-      attribute: 'hat',
-      value: true, 
-      category
+      currentQuestion = {
+        attribute: 'hat',
+        value: true, 
+        category
     } 
   } else if (category === 'glasses') {
-    currentQuestion = {
-      attribute: 'glasses',
-      value: true, 
-      category
-    }  
+      currentQuestion = {
+        attribute: 'glasses',
+        value: true, 
+        category
+      }  
```

Comment:
To make a bit less code, you can combine categories "glasses" and "hat" into single category called "accessories", and then :
`
currentQuestion = {
attribute: value,
value: true,
category: category,
}
`
Attribute has a property of the booleans such as glasses or hat.
In the "value" we are asking if this person wears a hat for example, so it always has to be true in the question.